projects/free-protocol/index.js

```diff
@@ -83,4 +83,9 @@ module.exports = {
       ]
     })
   },
+  kroma: {
+    tvl: sumTokensExport({
+      tokens: [ADDRESSES.kroma['M-BTC']]
```

Comment:
if nothing locked in kroma but MBTC is minted here, I would recommend changing this line to `tvl: () => ({})`

Reply:
I pushed new commit(#10947)[5712b92](https://github.com/DefiLlama/DefiLlama-Adapters/pull/10947/commits/5712b92639786892c2052402ae243a8548642a46) that applied your comment. Currently, 52 M-BTC already exist on Kroma Chain, having been minted. Additional M-BTC will be minted on Kroma Chain when Free Protocol bridges M-BTC from Merlin Chain to Kroma Chain.